plugins/python/plang/Environment.cpp

```diff
@@ -66,14 +66,37 @@ EnvironmentPtr Environment::get()
     return &g_environment;
 }
 
+
+#if (PY_VERSION_HEX >= 0x03000000)
+int wake_up_numpy()
+#else
+void wake_up_numpy()
+#endif
+{
+    import_array();
+#if (PY_VERSION_HEX >= 0x03000000)
+    return 0;
+#else
+#endif
+}
+
```

Comment:
Well, if nothing else we should move it back to Environment, because it's really unclear why it's here.  Should be able just to do: (void)import_array(), I would think and lose all the ifdef stuff.